packages/netlify-cms-backend-github/src/API.js

```diff
@@ -148,6 +148,14 @@ export default class API {
     });
   }
 
+  retrieveMetadataFile(key) {
```

Comment:
I originally planned to replace this with a query, but the GraphQL API doesn't return anything for `refs/meta` at the moment. I'm discussing it with GitHub support